components/FullEventCard/FullEventCard.js

```diff
@@ -27,22 +27,23 @@ export default function FullEventCard({ route }) {
 
       <View style={styles.lineItem}>
         <Text style={styles.label}>START TIME:</Text>
-        <Text style={styles.text}>{findDisplayTime(event.start_time)}</Text>
+        <Text style={styles.text}>{event.start_time}</Text>
+        {/* <Text style={styles.text}>{findDisplayTime(event.start_time)}</Text> */}
       </View>
 
       <View style={styles.lineItem}>
         <Text style={styles.label}>DURATION</Text>
-        <Text style={styles.text}>{event.duration} minutes</Text>
+        <Text style={styles.text}>{event.duration}</Text>
       </View>
 
       <View style={styles.lineItem}>
         <Text style={styles.label}># ATTENDING:</Text>
-        <Text style={styles.text}>{event.players_going}</Text>
+        <Text style={styles.text}>{event.current_participant_count}</Text>
       </View>
 
       <View style={styles.lineItem}>
         <Text style={styles.label}># REQUIRED:</Text>
-        <Text style={styles.text}>{event.max_players}</Text>
+        <Text style={styles.text}>{event.max_participant_count}</Text>
       </View>
 
```

Comment:
Thanks for fixing these keys. I understand your frustration with the change in data model. Way to refactor!